app/src/main/res/layout-w600dp/fragment_data_collection.xml

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
this is the layout file for displaying content on a tablet and mobile devices in landscape mode. The only difference here is the width of the bottomsheet.

Reply:
Just an idea - if the width is the only difference (and will be in the future), could the width just be a "variable" that is retrieved from the "value" resources depending on tablet vs mobile?

Reply:
Not sure if the width is the going to be the only difference in the future. This pattern seemed like the way to go for future changes/enhancement.

Reply:
Ok, that's fine 👍